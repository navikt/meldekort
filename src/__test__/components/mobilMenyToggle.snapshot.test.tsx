import * as React from 'react';
import * as renderer from 'react-test-renderer';
import ProviderWrapper from '../utils/providerWrapper';
import MobilMenyToggle from '../../app/components/meny/mobil/mobilMenyToggle';

it('Sprakvelger', () => {
  const tree = renderer.create(
    <ProviderWrapper>
      <MobilMenyToggle />
    </ProviderWrapper>
  );
  expect(tree).toMatchSnapshot();
});
