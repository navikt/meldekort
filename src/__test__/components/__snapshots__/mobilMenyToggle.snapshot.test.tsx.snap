// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MobilMenyToggle 1`] = `
<div
  className="burger-menu-toggle"
  onClick={[Function]}
>
  <div
    className="nav-icon open"
  >
    <span />
    <span />
    <span />
    <span />
  </div>
</div>
`;
